tests/generate_fixtures/generate_pyani_anim_matrices.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# The MIT License
+#
+# Copyright (c) 2024 University of Strathclyde
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# Make a temp subdir
+mkdir tmp_pyani
+cd tmp_pyani
+
+# Create pyanidb
+pyani createdb
+
+# Copy genomes to tmp_pyani subdir (macOS)
+for FASTA in ../../../tests/fixtures/viral_example/*.f*; do
+    ln -s $FASTA ${FASTA##*/};
```

Comment:
I just put the whole script through https://www.shellcheck.net/ (they have an online checker, I am wondering about adding this to our pre-commit setup - see #103), and it only had one issue to flag:

```
    ln -s "$FASTA" "${FASTA##*/}";
```

i.e. Double quote the filenames on the off chance they contain spaces.